user/models.py

```diff
@@ -44,7 +46,7 @@ def avatar(self):
         return images.first().image.url
 
     def save(self, *args, **kwargs):
-        if not self.images.filter(is_avatar=True).exists():
+        if self.pk and not self.images.filter(is_avatar=True).exists():
```

Comment:
Expanded the condition to handle the situation of the initial creation of the user